mkdocs.yml

```diff
@@ -5,6 +5,7 @@ site_url: "https://docs.csc.fi/"
 site_description: Instructions and user guides for the CSC supercomputers, cloud services, storage and software, including FAQ and tutorials
 
 extra:
+  environment: !ENV [MKDOCS_ENV, "production"] # FIXME to this: environment: !ENV [MKDOCS_ENV, "preview"]
```

Comment:
Comment still relevant?

Reply:
The comment is what I'll replace the key with momentarily.